web/webkit/SpecRunner.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Jasmine Spec Runner v2.4.0</title>
```

Comment:
I figured if a developer doesn't want to install node locally they can still run the tests through the browser. But, if you don't think that's necessary, I can remove it.